File: bddtests/src/main/kotlin/mocking/tpp/models/ViewPatientOverviewItem.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement
import javax.xml.bind.annotation.XmlValue

@XmlRootElement(name = "Item")
@XmlAccessorType(XmlAccessType.FIELD)
data class ViewPatientOverviewItem(
        @XmlAttribute var id: String = "default patientId",
        @XmlAttribute var description: String = "default description",
        @XmlAttribute var date: String = "default date",
        @XmlValue var value: String = "default value"
)

File: xamarinintegrationtests/NHSOnline.HttpMocks/Tpp/Models/ViewPatientOverViewItem.cs
using System.Xml.Serialization;

namespace NHSOnline.HttpMocks.Tpp.Models
{
    public class ViewPatientOverViewItem
    {
        [XmlAttribute("id")] public string? Id { get; set; }

        [XmlAttribute("description")] public string? Description { get; set; }

        [XmlAttribute("date")] public string? Date { get; set; }

        [XmlText] public string? Value { get; set; }
    }
}